server/.env

```diff
@@ -1,2 +1,3 @@
 TEAM_NAME="Shums"
 FLASK_ENV="development"
+Jason Chui
```

Comment:
So you'll want to add your name into the TEAM_NAME variable (inside the quotes)

Reply:
Got it.